cernopendata_client/printer.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client output print configuration."""
+
+import click
+
+
+def display_message(prefix=None, msg_type=None, msg=None):
```

Comment:
Please also hunt for the other occurrences of printing so that we centralise `==>` for all messages, e.g. we currently  have:

```console
$ cernopendata-client download-files

ERROR: Please provide at least one of following arguments: (recid, doi, title)
```

which is not consistent with:

```console
$ cernopendata-client verify-files --recid 5500 --server http://opendata.cern.ch
...
==> Verifying file demoanalyzer_cfg_level4MC.py...
  -> expected size 3874, found 3864
==> ERROR: File size does not match.
```

I would expect to see `==> ERROR` everywhere.

Reply:
Done 👍